API/src/app.ts

```diff
@@ -14,94 +14,144 @@ app.use(bodyParser.urlencoded({ extended: false }));
 app.use(bodyParser.json());
 const port = 3000;
 const JWT_SECRET: string = process.env.JWT_SECRET ?? "";
+const REFRESH_JWT_SECRET: string = process.env.REFRESH_JWT_SECRET ?? "";
+
 if (JWT_SECRET === "") {
   console.log("JWT_SECRET REQUIRED");
   process.exit(0);
 }
 
+if (REFRESH_JWT_SECRET === "") {
+  console.log("JWT_SECRET REQUIRED");
+  process.exit(0);
+}
+
 app.post("/register", async (req, res) => {
+  try {
```

Comment:
Just a knit pick, but my OCD driving me crazy that the code block isn't indented

Reply:
sry i need a js linter hahaha